docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -127,6 +127,28 @@ _jboss.server.config.dir_.
 If your directory is located outside of _jboss.server.config.dir_, then
 you need to change the _path_ and _relative-to_ values appropriately.
 
+It is possible to encrypt the filesystem realm by configuring the credential-store and secret-key attributes for the realm.
+
+When creating a an encrypted filesystem realm, you will have to specify two more attributes.
+
+* `credential-store`: This attribute specifies the credential-store where the secret key used for encrypting and decrypting the realm is stored.
+
+* `secret-key`: This attribute specifies the alias to the secret key stored in the credential store. By default the alias in the credential store is set to `key`
```

Comment:
s/alias to the secret key.../alias within the `credential-store` that contains the secret key to be used to encrypt and decrypt the realm

Reply:
s/By default the alias in the credential store is set to `key`/The default value is `key`.